From c897203a7f65a05e57c67e6cddee3f70110d0824 Mon Sep 17 00:00:00 2001
From: Jim Meyering <jim@meyering.net>
Date: Wed, 17 Oct 2012 16:55:02 +0200
Subject: [PATCH 41/89] maint: use $(AM_V_GEN) to cut down on build noise

* doc/Makefile.am: Prefix each rule with $(AM_V_GEN).
* doc/po4a.mk (dist_man_MANS): Likewise.
---
 doc/Makefile.am | 2 +-
 doc/po4a.mk     | 8 ++++----
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 08046d1..e773efa 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -12,6 +12,6 @@ EXTRA_DIST = FAT		\
 
 .PHONY: updatepo
 updatepo:
-	list='$(SUBDIRS)'; for dir in $$list; do \
+	$(AM_V_GEN)list='$(SUBDIRS)'; for dir in $$list; do \
 		$(MAKE) -C "$$dir" updatepo; \
 	done
diff --git a/doc/po4a.mk b/doc/po4a.mk
index b378121..aaf4024 100644
--- a/doc/po4a.mk
+++ b/doc/po4a.mk
@@ -41,13 +41,13 @@ install-man: install-man1 install-man5 install-man8
 
 # For each .po, try to generate the man page
 all-local:
-	for po in `ls -1 $(srcdir)/*.$(lang).po 2>/dev/null`; do \
+	$(AM_V_GEN)for po in `ls -1 $(srcdir)/*.$(lang).po 2>/dev/null`; do \
 		$(MAKE) $$(basename $${po%.$(lang).po}); \
 	done
 
 # Remove the man pages that were generated from a .po
 clean-local:
-	for po in `ls -1 $(srcdir)/*.$(lang).po 2>/dev/null`; do \
+	$(AM_V_GEN)for po in `ls -1 $(srcdir)/*.$(lang).po 2>/dev/null`; do \
 		rm -f $$(basename $${po%.$(lang).po}); \
 	done
 
@@ -55,7 +55,7 @@ clean-local:
 # Update the PO in srcdir, according to the POT in C.
 # Based on the gettext po/Makefile.in.in
 updatepo:
-	tmpdir=`pwd`; \
+	$(AM_V_GEN)tmpdir=`pwd`; \
 	cd $(srcdir); \
 	for po in *.$(lang).po; do \
 	  case "$$po" in '*'*) continue;; esac; \
@@ -83,7 +83,7 @@ dist-hook: updatepo
 
 # Build the pages
 partprobe.8:
-	for locale in pt_BR ; do \
+	$(AM_V_GEN)for locale in pt_BR ; do \
 		po4a-translate -f man -m $(srcdir)/../C/$@ -p $@.$$locale.po -l $@ $(po4a_translate_options) ; \
 		if [ -f $(srcdir)/$@.$$locale.po.addendum ]; then \
 			po4a-translate -f man -m $(srcdir)/../C/$@ -p $@.$$locale.po -l $@ -a $(srcdir)/$@.$$locale.po.addendum $(po4a_translate_options) ; \
-- 
1.8.5.3

